cmd/snap/cmd_changes.go

```diff
@@ -122,14 +125,55 @@ func (c *cmdChanges) Execute(args []string) error {
 
 func (c *cmdTasks) Execute([]string) error {
 	cli := Client()
-	return showChange(cli, c.Positional.ID)
+	var id changeID
+	if c.Positional.ID == "" && c.LastChangeType == "" {
```

Comment:
Thanks, good suggestion, I'm not used to this go-specific statements yet, but I like it. Done. I haven't however merged the two error case together, I think they should have separate error messages.